src/agent/io/sourcemapper.ts

```diff
@@ -108,9 +109,14 @@ function processSourcemap(
     }
 
     sources.forEach((src: string) => {
+      const key = path.normalize(path.join(prefix || '', src));
       infoMap.set(
-          path.normalize(src),
-          {outputFile: outputPath, mapFile: mapPath, mapConsumer: consumer});
+          key,
+          {
+            outputFile: path.normalize(path.join(prefix || '', outputPath)),
```

Comment:
`path.join` normalizes as well, so no need to `normalize` ourselves.

Here and elsewhere.

Reply:
Oh, ok I didn't know that.  I'll update this.